LoopKit/CarbKit/CarbStore.swift

```diff
@@ -206,7 +206,8 @@ public final class CarbStore: HealthKitSampleStore {
         calculationDelta: TimeInterval = 5 /* minutes */ * 60,
         effectDelay: TimeInterval = 10 /* minutes */ * 60,
         carbAbsorptionModel: CarbAbsorptionModel = .nonlinear,
-        provenanceIdentifier: String
+        provenanceIdentifier: String,
+        test_currentDate: Date? = nil
```

Comment:
```suggestion
        testCurrentDate: Date? = nil
```

Any reason for the mixed casing here?

Reply:
To signify it's "special", i.e. not part of the normal interface, but used for testing. It follows the convention set on the other stores.